oaknational/Oak-Web-Application
# Summary card component changes #480

## Description

- Changes how images are handled in component.
- Adds optional imageContainerProps

https://www.figma.com/file/lgns6S3DT4CbCqKQrEf0z4/Oak-Design-System-2022?node-id=939%3A154201 / https://www.figma.com/file/lgns6S3DT4CbCqKQrEf0z4/Oak-Design-System-2022?node-id=1368%3A152738

## Issue(s)

Fixes #480 

## How to test

1. Go to plan-a-lesson / curriculum pages
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
